modules/api-support-tiers-mapping.adoc

```diff
@@ -123,5 +123,28 @@ API groups that end with the suffix `monitoring.coreos.com` have the following m
 |`v1`
 |Tier 1
 
+|===
+endif::microshift[]
+
+ifndef::microshift[]
```

Comment:
Will check with MicroShift writers before merge on whether any special conditionals are needed, since I saw they were in use for other groups in this file.

Reply:
@ShaunaDiaz?

Reply:
Confirmed with Shauna via DM that this `ifndef` is good for now.